### PR TITLE
Added Node.js sample for events API

### PR DESCRIPTION
**Pull request by tstatler**
@lparis @yoko, would you mind testing this out? We could use this in the app token/events part of the API training. 

cc @jamiesmith (couldn't find Dean on github)

### CODE REVIEW

**Comment by lparis** [comment]
Running as job...I don't get any messages except "[stdout] BEARER_TOKEN environment variable is undefined. See README for details." There is no other log output either from APC or web console.

**Comment by tstatler** [comment]
Try uncommenting the first line in index.js and redeploying. This will
generate additional debut output.

// global.AUTOBAHN_DEBUG = AUTOBAHN_DEBUG = true;

I've seen this symptom before when the auth token wasn't getting included
in the request.

File: example-node-event-client/README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        apc service bind /apcera::http --job node-event-tester
+
+3. Create a text document named **event-tester.pol** that contains the following policy:
+
+        on job::/sandbox/admin::node-event-tester {
+          { permit issue }
+        }        
+
+        on job::/apcera {
+          if (auth_server@apcera.me->name == "job::/sandbox/admin::node-event-tester")
```
Comment (yhyakuna) [inline example-node-event-client/README.md]: I think the antecedent claim should be -->  `if(job fqnMatch "job::/sandbox/admin::node-event-tester")`

**Comment by yhyakuna** [comment]
LGTM

**Pull request merged**